packages/terra-toggle-section-header/src/ToggleSectionHeader.jsx

```diff
@@ -51,6 +56,7 @@ const defaultProps = {
   isAnimated: false,
   isInitiallyOpen: false,
   isTransparent: false,
+  isOpen: false,
```

Comment:
Is isOpen is passed in initially but isInitiallyOpen is not, should the toggle be open?

Reply:
With these changes, it will be open if isOpen is passed and isInitiallyOpen is not.I hope it is the correct workflow.